src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -130,7 +130,8 @@
    message])
 
 (defn create-validation-error-message [fails]
-  [:div {:style {:color (:exception-state colors)}}
+  [:div {:style {:color (:state-exception colors)
+                 :marginBottom "0.5rem"}}
```

Comment:
heh - we have `exception-state` in two other places in the codebase. (no need to hold this PR for that)